server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/SystemInfoPage.java

```diff
@@ -17,33 +17,30 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonarqube.pageobjects;
+package org.sonarqube.qa.util.pageobjects;
 
 import com.codeborne.selenide.CollectionCondition;
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Selenide;
 import com.codeborne.selenide.SelenideElement;
 
-import static com.codeborne.selenide.Condition.exist;
-import static com.codeborne.selenide.Condition.text;
-import static com.codeborne.selenide.Selenide.$;
-import static com.codeborne.selenide.Selenide.$$;
-
 public class SystemInfoPage {
   public SystemInfoPage() {
-    $(".page-title").should(exist).shouldHave(text("System Info"));
+    Selenide.$(".page-title").should(Condition.exist).shouldHave(Condition.text("System Info"));
   }
 
   public SystemInfoPage shouldHaveCard(String title) {
-    $$(".system-info-health-card-title").find(text(title)).should(exist);
+    Selenide.$$(".system-info-health-card-title").find(Condition.text(title)).should(Condition.exist);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal ".system-info-health-card-title" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)